driver/src/main/scala/api/collections/genericcollection.scala

```diff
@@ -1019,4 +955,80 @@ trait GenericCollection[P <: SerializationPack with Singleton] extends Collectio
       ()
     }
   }
+
+  // ---
+
+  final class AggregatorContext[T](
```

Comment:
Split to allow to proper encoding of the type dependencies (between `Cursor`, `CursorProducer` and `CursorFlattener`)